apps/console/src/features/applications/components/edit-application.tsx

```diff
@@ -257,17 +259,17 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
      * Fetch the allowed origins list whenever there's an update.
      */
     useEffect(() => {
-        const allowedCORSOrigins = [];
+        const allowedCORSOrigins: string[]= [];
 
         if (OrganizationUtils.isCurrentOrganizationRoot()) {
             getCORSOrigins()
                 .then((response: CORSOriginsListInterface[]) => {
-                    response.map((origin) => {
+                    response.map((origin: CORSOriginsListInterface) => {
                         allowedCORSOrigins.push(origin.url);
                     });
                     setAllowedOrigins(allowedCORSOrigins);
                 })
-                .catch((error) => {
+                .catch((error: any) => {
```

Comment:
Let's use AxiosError here

Reply:
Addressed in 5be7c09df5ecb1b06d711304eeea6e6faf90d105